content/geoip/release-notes/2023.mdx

```diff
@@ -7,6 +7,30 @@ title: GeoIP2 Release Notes
 Subscribe to the [GeoIP2 release notes RSS feed](/geoip/release-notes/rss.xml).
 </Alert>
 
+<ReleaseNote date="2023-04-10" title="GeoNames Monthly Diff Report April 2023">
+[GeoNames Monthly Diff Report (April 2023)](/csv-files/GeoNames-Monthly-Diff-Report-April-2023.csv)
+
+GeoIP2 and GeoLite2 databases incorporate
+[GeoNames geographical data](https://www.geonames.org/), which is made available
+under the
+[Creative Commons Attribution 4.0 License](https://creativecommons.org/licenses/by/4.0/).
+
+Monthly updates to GeoNames data are incorporated into our databases as we
+refresh them. Above you will find a CSV report of these changes for the month of
+April as they relate to the following fields:
+
+- `geoname_id`
+- `country_name`
+- `country_code`
+- `city_name`
+- `subdivision_1_iso_code`
+- `subdivision_2_iso_code`
+
+Note that the values for the 'old' and ‘new’ columns for a given field only
```

Comment:
It looks like old may need backtick quotes instead of single-quote

Reply:
oh, i see. new isn't actually using backticks, but it is using a different glyph from the ' ...